### DISCUSSION TITLE
Feature request: SMEA sampler

### DISCUSSION THREAD

**Discussion by CasanovaSan** [lllyasviel/stable-diffusion-webui-forge#2344] [Ideas]
Can this sampler be added please?
https://civitai.com/articles/4856/a-new-sampler-to-generate-better-hands

**Comment by MisterChief95** [comment]
I made an extension that adds these samplers + Euler Max to Forge. They work fine for base image generation, but tend to show issues in img2img scenarios like hires fix or adetailer. Euler Max seems to work fine all the time though.

https://github.com/MisterChief95/sd-forge-extra-euler-samplers

**Reply by Squibeel** [reply]
> I made an extension that adds these samplers + Euler Max to Forge. They work fine for base image generation, but tend to show issues in img2img scenarios like hires fix or adetailer. Euler Max seems to work fine all the time though.
> 
> https://github.com/MisterChief95/sd-forge-extra-euler-samplers

First of all thanks, I'm using your extension.
Would you be able to do the same with the CFG++ samplers?

**Reply by MisterChief95** [reply]
@DenOfEquity actually already did! They have an extra schedulers extension here:
https://github.com/DenOfEquity/webUI_ExtraSchedulers
